faq/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FAQ
+---
```

Comment:
Generally, from my experience, the best practice is to forgo `permalink` unless strictly necessary because it represents a duplication of data. For example, this page currently exists at `/faq/` already.

If you'd like for me to add permalinks though, I can.